src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: $card-size;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+    margin-top: 40px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $second-text-color;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    &-reviews {
+      font-weight: 400;
+      font-size: 10px;
+      color: $main-text-color;
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      color: $second-text-color;
+      display: flex;
+      align-items: center;
+    }
+
+    &-amount {
+      font-weight: 700;
+      font-size: 16px;
+      color: $main-text-color;
+    }
+  }
+  &__btn {
+    width: 166px;
+    height: 40px;
+    background: $button-color;
+    border: none;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    color: #fff;
+    margin: 0 auto;
+    text-align: center;
+    margin-top: 16px;
+    transition: all 0.5s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated